src/lib.rs

```diff
@@ -8,6 +8,8 @@
 //! address, which is also shielded).
 
 #![cfg_attr(docsrs, feature(doc_cfg))]
+// Temporary until we have more of the crate implemented.
+#![allow(dead_code)]
```

Comment:
This change silences almost all the spurious warnings. The remainder are addressed by #44.